bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml

```
@@ -91,6 +91,7 @@ data:
             "setuid",
             "sigaltstack",
             "socket",
+            "stat",           # Adding 'stat' to support mkdirall functionality
```

```suggestion
            "stat", 
```
Adding 'stat' to support mkdirall functionality